indra-common/src/main/kotlin/net/kyori/indra/IndraExtension.kt

```diff
@@ -117,6 +117,13 @@ open class IndraExtension @Inject constructor(objects: ObjectFactory) {
     url = "https://opensource.org/licenses/Apache-2.0"
   ))
 
+  fun gpl30OnlyLicense() = this.license.set(License(
+    spdx = "GPL-3.0-only",
+    name = "GNU General Public License version 3",
+    url = "https://opensource.org/licenses/GPL-3.0",
+    bintray = "GPL-3.0"
```

Comment:
Bintray is silly and does not use SPDX identifiers: https://www.jfrog.com/confluence/display/BT/Bintray+REST+API#BintrayRESTAPI-DeleteDockerTag